ghdocs/Tutorial.md

```diff
@@ -0,0 +1,153 @@
+
+## Step 3: Add DocumentCardBasic component
+Open the `documentcardbasic-demo` project folder in Visual Studio Code.
+
+Open the file `src\App.js` and replace the existing code inside `render` method to only return the `DocumentCard` component:
+
+```jsx
+<div>
+  <DocumentCard onClickHref='http://bing.com'>
+      <DocumentCardPreview
+        previewImages={ [
+          {
+            previewImageSrc: require('./documentpreview.png'),
+            iconSrc: require('./iconppt.png'),
+            width: 318,
+            height: 196,
+            accentColor: '#ce4b1f'
+          }
+        ] }
+      />
+      <DocumentCardTitle title='Revenue stream proposal fiscal year 2016 version02.pptx'/>
+      <DocumentCardActivity
+        activity='Created Feb 23, 2016'
+        people={
+          [
+            { name: 'Kat Larrson', profileImageSrc: require('./avatarkat.png') }
+          ]
+        }
+        />
+    </DocumentCard>
+</div>
+```
+`DocumentCard` component contains the `DocumentCardPreview`, `DocumentCardTitle` and `DocumentCardActivity` components that help you describe the document card.
+
+Since office-ui-fabric-react is an npm module, we will need to import the document card component in the `App` component.
+
+Add the following import statement at the top of the `src\App.js` file just below `import './App.css'`:
+
+```js
+import {
+  DocumentCard,
+  DocumentCardPreview,
+  DocumentCardTitle,
+  DocumentCardActivity
+} from 'office-ui-fabric-react/lib/DocumentCard';
+```
+
+Notice how we import only the specific component module from `lib` folder instead of importing the entire `office-ui-fabric-react` module.
+
+Save the `src\App.js` file.
+
+Below is an example of how you can similarly import other components:
+
+```js
+import { Button } from 'office-ui-fabric-react/lib/Button';
+```
+
+Open `index.html` file.
+
+Office UI Fabric react components provide support for Left-to-Right (LTR) and Right-to-Left (RTL) rendering out of the box.
+
+You can specify the default direction in your `index.html`. Add the `dir` attribute with the value `ltr` to the `<html>` element:
+
+```html
+<html lang="en" dir="ltr">
+```
+
+Load Office UI Fabric styles by linking to the Office UI Fabric CDN. Add the following to the `<head`> element:
+
+```html
+<link rel="stylesheet" href="https://appsforoffice.microsoft.com/fabric/2.2.0/fabric.min.css">
+```
+
+Save the file.
+
+Copy the following images from GitHub to `src` folder.
+* [avatarkat.png](./images/avatarkat.png)
+* [iconppt.png](./images/iconppt.png)
+* [documentpreview.png](./images/documentpreview.png)
+
+## Step 4: Preview document card
+Switch to the console and type the following to preview your app in the browser:
+
+```
+npm start
+```
+
+You should see the document card component in your app.
+
+![DocumentCard preview](./images/react-app-doccard-preview.png)
+
+## Next steps
+Congratulations! You have successfully built a simple react app with an Office UI Fabric React component.
```

Comment:
2 final notes:
- Would you mind capitalizing the file name (to make it consistent with the other md files?
- Please link to this from the Get started section on the main README.md - we want everyone to be aware of this goodness :-)